test/e2e/create_test.go

```diff
@@ -139,14 +158,24 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 		t.Skip("BYOH testing disabled")
 	}
 
-	err := tc.disableClusterMachineApprover()
-	require.NoError(t, err, "failed to scale down Machine Approver pods")
-
-	err = tc.provisionBYOHConfigMapWithMachineSet()
-	require.NoError(t, err, "error provisioning BYOH ConfigMap with MachineSets")
-
+	// For platform-agnostic infrastructure just validate the BYOH ConfigMap
+	// TODO: See https://github.com/openshift/windows-machine-config-operator/pull/858#discussion_r780316359
+	if tc.CloudProvider.GetType() == config.NonePlatformType {
+		err := tc.validateWindowsInstanceConfigMap(int(gc.numberOfBYOHNodes))
+		if err != nil {
+			require.NoError(t, err, "error validating windows-instances ConfigMap")
+		}
```

Comment:
unneeded if statement, just the require.NoError is sufficient

Reply:
Good catch. Fixed in https://github.com/openshift/windows-machine-config-operator/commit/a6d72ad9f28b382f64f31a8d34cea5a3ce57035e